pkgs/kde/gear/spectacle/default.nix

```diff
@@ -7,6 +7,13 @@
 mkKdeDerivation {
   pname = "spectacle";
 
-  extraBuildInputs = [qtwayland qtmultimedia opencv];
+  extraBuildInputs = [
+    qtwayland
+    qtmultimedia
+    # https://invent.kde.org/graphics/spectacle/-/blob/master/CMakeLists.txt?ref_type=heads#L83
+    ((opencv.override { runAccuracyTests = false; }).overrideAttrs ({ cmakeFlags, ... }: {
+      cmakeFlags = cmakeFlags ++ [ "-DBUILD_LIST=core,imgproc" ];
```

Comment:
Can we maybe get a real parameter on the opencv package for this?

Reply:
That's a good idea. Then also the override will be more minimal.